vaadin-dev-server/src/main/resources/com/vaadin/base/devserver/dev-mode-not-ready.html

```diff
@@ -54,7 +54,7 @@
 const poll = () => {
   try {
     fetch(window.location.href, { headers: { 'X-DevModePoll': 'true' } }).then(response => {
-      if (response.headers.has("X-DevModePending")) {
+      if (!response || !response.headers || response.headers.has("X-DevModePending")) {
```

Comment:
Seems superficial, `fetch` must resolve to `Response` as per spec.

```suggestion
      if (response.headers.has("X-DevModePending")) {
```

Reply:
Not sure what the case was there `then` was called with an undefined `response` but maybe it is not important